Source/WebCore/css/FontFaceSet.cpp

```diff
@@ -261,7 +261,7 @@ void FontFaceSet::faceFinished(CSSFontFace& face, CSSFontFace::Status newStatus)
 
 void FontFaceSet::startedLoading()
 {
-    // FIXME: Fire a "loading" event asynchronously.
```

Comment:
Not sure how much it matters, but your code fires the event synchronously (which may be correct depending on what the comment is relative to)